synapse/lib/certdir.py

```diff
@@ -100,6 +101,15 @@ def _saveCertTo(self, cert, *paths):
             fd.write(crypto.dump_certificate(crypto.FILETYPE_PEM, cert))
         return path
 
+    def _saveClientCertTo(self, cert, *paths):
```

Comment:
Would it be better to combine with `_SaveCertTo` and pull out the bytes by the cert type?